### ISSUE TITLE
[docs] Please document how to generate new entries that are not tied to an existing issue/PR

### ISSUE THREAD

**Issue by offbyone** [twisted/towncrier#365]
I don't want to have the bookkeeping requirement of creating a github issue for everything I mention in the changelog... but I'm not sure what those files need to look like.

**Comment by adiroiban** [comment]
Hi. I don't understand the issue.

The towncrier release notes "fragments" are not tied to GitHub issues/PR

You can create fragments using Jira/Trac or any system.

-------

A release announcement has the release notes groped into various categories.

To help with support and make it easy to find more detatils about a certain released featured or bug, there is a issue/ticket ID associated.

The category and ticket ID meta data are encoded in the release fragment file name.

I guess that you want to create "fragments" with just the category marker and without the issue marker.

I don' think that this is a common usecase and is not supported for now.
Feel free to send a PR.

One workaround is to use '0' as the ticket id.
Then you can have a custom template that will skip the link for ticket id 0, but include the release fragment content.

Hope it helps

**Comment by offbyone** [comment]
My question is, how to create fragments that are not tied to _any_ issue tracker. Suppose I'm just working on code for myself, where I want the changelog, but I don't use an issue tracker. How do I generate fragments for that?

**Comment by adiroiban** [comment]
maybe something like this?

You will still need a "code" for each release note item.


you can configure the output not to include a link 

```diff
diff --git a/pyproject.toml b/pyproject.toml
index df86fa4..df7096b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -2,7 +2,7 @@
     package = "towncrier"
     package_dir = "src"
     filename = "NEWS.rst"
-    issue_format = "`#{issue} <https://github.com/hawkowl/towncrier/issues/{issue}>`_"
+    issue_format = "{issue}"
```

have a file like

```
$ cat src/towncrier/newsfragments/SOME-FEATURE.removal.rst
Some text for this new feature to be released
```

the result is  like

```
$ towncrier build --draft
Loading template...
Finding news fragments...
Rendering news fragments...
Draft only -- nothing has been written.
What is seen below is what would be written.

towncrier 21.9.0 (2022-06-10)
=============================

Deprecations and Removals
-------------------------

- Some test (SOME-FEATURE)
```

**Comment by SmileyChris** [comment]
I agree that it would be nice to be able to have fragments that aren't tied to features/issues at all. Even in a project which uses an issue tracker, as a developer I may just want to add some docs / a quick code improvement that doesn't have an issue assigned.

My thoughts are using some prefix (like maybe `+`) to identify a change that isn't tied to any issue/feature (the rest of the base filename can be any random string -- `towncrier create +.feature` could even append a random hash to the base name).

**Comment by adiroiban** [comment]
Note that for `towncrier` each fragment is a filename.

So it needs a unique name.

```

You can name it something like `dev-docs-update.docs` ... or `BRANCH_NAME.feature`

And then create your own template.

-------

But I think that this disucssion need more detaisl.

What is the required input

What is the expected output

What is the actual/current output

**Comment by SmileyChris** [comment]
I understand that. I still want fragments tied to issues. I just *also* want to be able to create fragments that are not tied to an issues. I'm just throwing up a PR now as a (fully working and tested) possible implementation...

**Comment by adiroiban** [comment]
This should be fixed by #428

Please comment if you think that more work is needed.
